unlock-app/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
```

Comment:
This line allows us to have a hybrid JS/TS project. If we get to 100% TS in the future, we should set this to `false` to avoid the introduction of any additional JS.